test/clj/harja/testi.clj

```diff
@@ -816,10 +822,18 @@
                    FROM   lupaus_sitoutuminen
                    WHERE  \"urakka-id\" = (SELECT id FROM urakka where nimi = 'Iin MHU 2021-2026')"))))
 
-(defn hae-oulun-maanteiden-hoitourakan-2019-2024-sopimus-id []
+
+(defn hae-raahen-maanteiden-hoitourakan-2023-2028-id []
```

Comment:
Oon pyrkinyt itse kohti tän funktion käyttöä:
(hae-urakan-id-nimella "YHA-päällystysurakka") eli että käytettäis aina mahdollisuuksien mukaan tuota, eikä olisi joka urakalle omia funktioita.